sklearn/utils/validation.py

```diff
@@ -297,6 +300,10 @@ def _ensure_sparse_format(spmatrix, accept_sparse, dtype, copy,
     if isinstance(accept_sparse, six.string_types):
         accept_sparse = [accept_sparse]
 
+    if LooseVersion(scipy_version) < '0.14.0':
```

Comment:
I think you should do this check once when the module is loaded and store a `LARGE_SPARSE_SUPPORTED` global variable.

Reply:
Do you understand what I mean here?